docs/usage/modules/datasource.md

```diff
@@ -1,7 +1,7 @@
 # Datasources
 
-Once Renovate's Managers are done scanning files and extracting dependencies, they will assign a `datasource` to each so that Renovate then knows how to search for new versions.
-You do not need to ever configure/override Datasources directly, but you may use them in `packageRules` to configure other aspects of Renovate's behavior, e.g.
+Once Renovate's manager is done scanning files and extracting dependencies, it will assign a `datasource` to each (what? dependency maybe?) so that Renovate then knows how to search for new versions.
```

Comment:
I'm not sure what the "will assign a `datasource` to" part is referring to?
The current text is not clear enough for me to understand what the `datasource` is being added to.